### PR TITLE
[docs] PR for Document PLAT-1702 (Doc-PLAT-2397)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2397

Preview: https://deploy-preview-10781--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md [hunk 1]
```diff
@@ -65,118 +65,129 @@ showAsideToc: true
...
+The preceding setting specifies the resource group as `different-rg` and the DNS zone as `dns.example.com`. The `different-rg` resource group in the resource ID overrides the setting in the **Resource Group** field, as shown in the following illustration:
 
-1. Add a link to the virtual network that you want it to be connected to. (For more details, refer to [this Microsoft document](https://docs.microsoft.com/en-us/azure/dns/private-dns-getstarted-portal).)
+![img](/images/yb-platform/install/azure/private-dns-myrg-2.png)
 
-1. To use the Private DNS Zone in Platform, add either the Resource ID OR the name of the DNS zone to the “Private DNS Zone” field.
+## Specify region
 
-    <br/><br/>
+You can specify a region as follows:
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md:150]: other sections have numbered steps

**Review by ddhodge — APPROVED** [review]
minor comments

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md [hunk 2]
```diff
@@ -65,118 +65,129 @@ showAsideToc: true
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md:65]: changed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md [hunk 3]
```diff
@@ -65,118 +65,129 @@ showAsideToc: true
...
-![Description](/images/yb-platform/install/azure/private-dns-different-rg.png)
+Typically, it takes a few minutes for the cloud provider to be configured. When the configuration completes, you will be ready to create a YugabyteDB universe on Azure.
 
-## Specify Region Info
+### How to use a shared image gallery
 
-* Select a region and provide a virtual network name from your Azure portal.
-* Security group is only needed if the database VM is in a different network than the platform.
-* Provide the mapping of what Subnet IDs to use for each Availability Zone you wish to be able to deploy. This is required to ensure the Yugabyte Platform can deploy nodes in the correct network isolation that you desire in your environment.
+You can use shared image galleries as an alternative to using marketplace image URNs. A gallery allows you can provide your own custom image to use for creating universe instances. For more information on shared image galleries, refer to [Store and share images in an Azure Compute Gallery](https://docs.microsoft.com/en-us/azure/virtual-machines/shared-image-galleries).
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md:170]: fixed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md:150]: changed

**Review by haikarthikssk — APPROVED** [review]

**Comment by lizayugabyte** [comment]
Will merge the PR on Dec 15 for 2.11.1

**Pull request merged**